### PR TITLE
Add state and fixer to specify that onboarding in over

### PR DESCRIPTION
**Pull request by jinroh**

### CODE REVIEW

File: web/jobs/jobs.go [hunk 1]
```diff
@@ -287,6 +287,9 @@ func getJob(c echo.Context) error {
 
 func cleanJobs(c echo.Context) error {
 	instance := middlewares.GetInstance(c)
+	if err := permissions.AllowWholeType(c, permissions.GET, consts.Jobs); err != nil {
+		return err
+	}
```
Comment (nono) [inline web/jobs/jobs.go]: Not sure why it is in this PR, but OK :+1:

File: web/settings/instance.go [hunk 2]
```diff
@@ -43,6 +43,7 @@ func getInstance(c echo.Context) error {
 		return err
 	}
 	doc.M["locale"] = instance.Locale
+	doc.M["onboarding_finished"] = instance.OnboardingFinished
```
Comment (nono) [inline web/settings/instance.go]: We should also modify the `updateInstance` below

Comment (jinroh) [inline web/settings/instance.go]: I was hesitant to allow mutation from this endpoint and decided to set the flag on the `/context/onboarded` handler which provide the redirect at the end of the onboarding: see https://github.com/cozy/cozy-stack/pull/894/files#diff-0ff4658f63295d93dc43ccf67d16d07cR43

Comment (nono) [inline web/settings/instance.go]: OK. Let's fix the tests and merge as is

**Pull request merged**